gdal/frmts/ceos2/sar_ceosdataset.cpp

```diff
@@ -1513,29 +1563,60 @@ int SAR_CEOSDataset::ScanForMapProjection()
 
     for( int i = 0; i < nGCPCount; i++ )
     {
-        char         szId[32];
+        char         szId[34];
```

Comment:
I don't think this change is needed
```suggestion
        char         szId[32];
```